tests/unit/api/test_feature_list.py

```diff
@@ -1093,14 +1089,19 @@ def test_get_feature_jobs_status_feature_without_tile(
     """
     Test get_feature_jobs_status for feature without tile
     """
-    mock_execute_query.return_value = feature_job_logs[:0]
     saved_scd_table["col_text"].as_entity(cust_id_entity.name)
 
     scd_view = saved_scd_table.get_view()
     feature = scd_view["effective_timestamp"].as_feature("Latest Record Change Date")
     feature_list = FeatureList([feature, float_feature], name="FeatureList")
     feature_list.save()
-    job_status_result = feature_list.get_feature_jobs_status()
+
+    with patch(
+        "featurebyte.session.snowflake.SnowflakeSession.execute_query"
+    ) as mock_execute_query:
+        mock_execute_query.return_value = feature_job_logs[:0]
+        job_status_result = feature_list.get_feature_jobs_status()
+
```

Comment:
Hit the flaky test locally when calling `featurebyte.session.snowflake.SnowflakeSession.list_databases` after `featurebyte.session.snowflake.SnowflakeSession.execute_query` is patched by the decorator. Change the patch decorator to patch context manager so that it won't interfere with the `snowflake_execute_query` at `tests/unit/conftest.py`.